src/components/new-request/search.vue

```diff
@@ -136,18 +146,19 @@
           <span>{{ entityConversionText }}</span>
         </v-tooltip>
       </v-col>
-    </v-row>
 
-    <v-row no-gutters>
-      <!-- jurisdiction for xpro/mras -->
-      <v-col cols="12" md="4" lg="4" v-if="getIsXproMras">
-        <v-select :error-messages="getErrors.includes('jurisdiction') ? 'Please select a jurisdiction' : ''"
-                  :items="jurisdictionOptions"
-                  :menu-props="{ bottom: true, offsetY: true}"
-                  label="Select business' home jurisdiction"
-                  @change="clearErrors()"
-                  filled
-                  v-model="jurisdiction">
+      <!-- Jurisdiction for xpro/mras -->
+      <v-col v-if="getIsXproMras" cols="12" md="6" class="py-0">
```

Comment:
I put this column in the same row as above, which I think matches the uxpin better. Anyway, this can be tweaked when we work on the extrapro flow.